script/Ciel/Day 8/08_01_CIEL08_1B.txt

```diff
@@ -113,22 +126,30 @@ That being said, I'm no pro. I don't normally use it much, but there's something
 -- Page 43, Offset 18369.
 -- 「そりゃあ時間の許す限りは。一行や二行の言葉のやりとりで分かるのは一行二行分のコトだけだし。連絡事項としては便利だけど、会話をするなら会って話した方が断然いい」
 // not only does this sound dumb but its also bullshit
-"Well yeah, if I have the time for it. By writing a one or two line message, only the message itself is conveyed. It's usefel for relaying information, but if I want to have a proper conversation it'd be much better to meet in person."
+// ok :b:oomer
+"Well, so long as I've got the time.
+ Texting someone a sentence or two can only ever convey a sentence or two worth of information.
+ It's fine for just giving someone a heads up, but if you want to have a proper conversation it's way better to meet up and have a chat."
 }
 [sha:9773650c50ef78e8da3a86e73a6cdd024bfdb39f]{
 -- Page 44, Offset 18370.
 -- 「一部分しか表せない<連絡|システム>なのに、その一部分を見た人間はそれが相手の全部分だと錯覚する、という事ですね。
-"Though they are faced with a system of communication that only expresses one side, people incorrectly believe they've seen the whole picture―is what you're saying.
+// _相手_の全部分 kinda split across the comma
+"So you're saying that even though it's a method of communication through which people can only express a small part of themselves,
+ people reading it incorrectly believe they've seen the whole picture?
 }
 [sha:c2d184a596e18570386f6f8dba345a453f380f43]{
 -- Page 44, Offset 18371.
 -- たとえば、発信者は爪先しか見せていないのに、その爪が猫のものだっただけで全身まで猫だ、と決めつけるような。
-It's like seeing just the tips of an animal's toes, thinking they resemble those of a cat, and deciding the animal must be a cat.
+// On the internet, nobody knows you're a cat
+To give an analogy, it would be like seeing the tips of someone's toes,
+ thinking that their nails resemble the claws of a cat,
+ and so deciding that the rest of their body must also resemble that of a cat.
 }
 [sha:6272cfdd3adbf165436fabe1a0d1ee78e2545be2]{
 -- Page 44, Offset 18372.
 -- 私もその意見には賛成です。文字だけならいくらでも嘘をつけるのが人間ですし」
-I am in complete agreement. People can lie as much as they want with writings."
+I am in complete agreement. People lie in print as much as they like."
```

Comment:
Something like 'It's very much like humans to think they can get away with lying just because they're communicating with letters' is probably more what's trying to be said here.

The print part doesn't fit too--she's talking about written communication generally.